### ISSUE TITLE
Seed does not work in combination with {min, max} quantifier

### ISSUE THREAD

**Issue by PascalSchumacher** [mifmif/Generex#24]
Thanks again for adding the seed. :+1: 

It works fine for most cases, but fails for when a regular expression with a {min, max} quantifier is used.

Example:

```java
@Test
public void testSeedWithMinMaxQuantifier() {
    long seed = -5106534569952410475L;
    String pattern = "[A-Z]{1,10}";
    
    Generex firstGenerex = new Generex(pattern);
    firstGenerex.setSeed(seed);
    String firstValue = firstGenerex.random();
    
    Generex secondGenerex = new Generex(pattern);
    secondGenerex.setSeed(seed);
    String secondValue = secondGenerex.random();
    
    Assert.assertEquals(firstValue, secondValue);
}
```

It works for exact quantifiers.

**Comment by mifmif** [comment]
fixed in version 1.0.1

**Comment by PascalSchumacher** [comment]
Thanks! :smile:
